DependencyInjection/Configuration.php

```diff
@@ -70,8 +70,8 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                     ->prototype('array')
                         ->children()
                             ->scalarNode('subject')->isRequired()->cannotBeEmpty()->end()
-                            ->scalarNode('attribute')->isRequired()->defaultValue(Attribute::class)->cannotBeEmpty()->end()
-                            ->scalarNode('option')->isRequired()->defaultValue(Option::class)->cannotBeEmpty()->end()
+                            ->scalarNode('attribute')->isRequired()->defaultValue(ProductAttribute::class)->cannotBeEmpty()->end()
+                            ->scalarNode('option')->isRequired()->defaultValue(ProductOption::class)->cannotBeEmpty()->end()
```

Comment:
In that case I'll keep the dependency with Sylius Product 👍

However one of the reasons for creating Archetype initially was to decouple from Product.

https://github.com/Sylius/Sylius/pull/2225
https://github.com/Sylius/Sylius/issues/2001